src/pages/Swap.tsx

```diff
@@ -131,21 +131,26 @@ function Swap(): ReactElement {
       .filter(({ isLPToken }) => !isLPToken)
       .filter(({ symbol }) => {
         // get list of pools containing the token
-        const tokenPools = tokenToPoolsMap[symbol] ?? []
+        if (!tokenSymbolToPoolNameMap[symbol]) return false
+        const tokenPools = tokenSymbolToPoolNameMap[symbol]
         // ensure at least one pool is unpaused to include token in swappable list
         const hasAnyUnpaused = tokenPools.some((poolName) => {
-          const basicPool = basicPools?.[poolName]
-          return basicPool ? !basicPool.isPaused : false
+          if (!basicPools) return false
```

Comment:
Could combine this to
``` 
if (basicPools == null || basicPools[poolName] == null) return false
return !basicPool[poolName].isPaused`
```
Since the right hand side won't run if the first condition is true.

Reply:
Errs w\ basicPool could be undefined.